.dockerignore

```diff
@@ -0,0 +1,25 @@
+*.pyc
+__pycache__
+.git/
+.github/
+.gitignore
+.travis.yml
+CODE_OF_CONDUCT.md
+CONTRIBUTING.rst
+Dockerfile
+LICENSE.rst
+MANIFEST.in
+README.rst
+docker-compose.yml
+docs/
+pre-commit-hook*
+/.coverage
+.DS_Store
+/.mypy_cache
+/.pytest_cache
+/.tox
+*.egg-info
+/htmlcov
+/dist
+/build
+/.venv.egg-info/
```

Comment:
Woulid it be easier to ignore `*` and then unignore things we want to keep (with `!`) ?

Reply:
I am not sure what we want to ignore or keep. As it's only a development docker, we could just keep everything.

Reply:
As discussed with @elemoine , I removed the `.dockerignore` file and only copied the files necessary for the pip install in Dockerfile.

As we later mount the whole repository anyway, we can only including what is strictly necessary for install in the image.